src/blocks/HeaderBlockBrandPivot/MenuItem.tsx

```diff
@@ -72,6 +82,24 @@ const MenuLink = styled('a')({
     paddingLeft: '1.5rem',
   },
 })
+
+const MenuGroupLabel = styled('span')({
+  display: 'block',
+  color: colorsV3.gray500,
+  textTransform: 'uppercase',
+  fontSize: '0.8rem',
+  lineHeight: '1rem',
+  whiteSpace: 'nowrap',
+  padding: 0,
```

Comment:
The span doesn't have any default padding by default that we need to reset right? 
```suggestion
```